src/pages/stats/index.tsx

```diff
@@ -1,15 +1,128 @@
+import { supabaseServiceRole } from "@lib/supabaseServiceRole";
+import { createServerSupabaseClient } from "@supabase/auth-helpers-nextjs";
+import { GetServerSidePropsContext } from "next";
 import Head from "next/head";
 import PageLayout from "src/components-client/PageLayout";
 
-export default function Home() {
+function Block({ value, label }: { value: string | number; label: string }) {
+  return (
+    <>
+      <div className="flex aspect-square flex-col items-center justify-center border border-black text-center">
+        <p className="pb-2 text-7xl font-black">{value}</p>
+        <h3 className="text-3xl ">{label}</h3>
+      </div>
+    </>
+  );
+}
+
+interface Props {
+  data: {
+    player: {
+      playerName: string;
+      gamesPlayed: number;
+      gamesWon: number;
+      pointsScored: number;
+    };
+    office: {
+      totalGames: number;
+      totalRallies: number;
+    };
+  };
+}
+
+export default function Home({
+  data: {
+    player: { gamesPlayed, gamesWon, playerName, pointsScored },
+    office: { totalGames, totalRallies },
+  },
+}: Props) {
   return (
     <>
       <Head>
         <title>Stats • Vincit Table Tennis</title>
       </Head>
       <PageLayout segment="STATS :D">
-        <h1 className="bg-red-200 text-3xl font-bold">Hello world!</h1>
+        <div className="flex flex-col gap-8">
+          <section>
+            <h2 className="pb-8 text-3xl">Personal ({playerName})</h2>
```

Comment:
Sorry this is a nitpick, but can you also change `pb` here to 4 or 6? I think it looks a little high.